AssetInformationApi/V1/Infrastructure/AddRepairsContractsToAssetEventConstants.cs

```diff
@@ -0,0 +1,11 @@
+namespace AssetInformationApi.V1.Infrastructure
+{
+    public static class AddRepairsContractsToAssetEventConstants
+    {
+        // JWT TOKEN
+        public const string V1_VERSION = "v1";
+        public const string EVENTTYPE = "AddRepairsContractsToAssetEvent";
+        public const string SOURCE_DOMAIN = "Asset";
```

Comment:
Is this SOURCE_DOMAIN correct? These properties were copied from the CreateAssetEvent, with the only difference being the EVENT_TYPE

Reply:
This is correct